osrf/vrx-docker
# Fix var_wind_time

These changes reconcile `var_wind_time` values with our wind envelop in the competition documentation, which says the minimum value will be 2. The purpose of this change is just to avoid confusion about allowed values. The example worlds in `vrx_gazebo/worlds/2022_practice` have already been updated, so this PR is just making the corresponding `.yaml` files match.

Since `generate_worlds.py` is broken, this needs to be tested using the `M1chaelM/generate_worlds` branch of `vrx`.

To test:
1. Switch to the correct branch of `vrx`:
    ```
    cd ~/vrx_ws/src/vrx
    git fetch
    git checkout M1chaelM/generate_worlds
    ```
1. Switch back to the vrx-docker directory:
    ```
    cd ~/vrx_ws/src/vrx-docker
    ```
3. Generate all the worlds:
    ```
    ./multi_scripts/prepare_all_task_trials.bash
    ```
4. Make sure the output matches the contents of `vrx_gazebo/worlds/2022_practice`:
    ```
    cd generated/task_generated
    for folder in *; do for world in $folder/worlds/*.world; do diff $world ~/vrx_ws/src/vrx/vrx_gazebo/worlds/2022_practice/${world##*/}; done; done

    ```
The output should only show a difference for each file where the autogeneration comment was removed. For example:
```
< 
< 
< 
< 
```
If this is the only difference, then everything is working.
